src/ngx_http_lua_kong_module.c

```diff
@@ -305,3 +305,172 @@ ngx_http_lua_kong_ffi_get_full_client_certificate_chain(ngx_http_request_t *r,
     return NGX_ABORT;
 #endif
 }
+
+
+#if (NGX_HTTP_SSL)
```

Comment:
It was my understanding that you wanted to get rid of those? Nonetheless, if we do decide to keep them around, the other FFI C functions in this module have safeguards for when NGINX is compiled without SSL support. Or were those safeguards what you wanted to get rid of (and not the macro guards)?

Reply:
I do intend to remove them. But for the scope of this feature I want to keep the PR more readable so they were left in. I would have a separate PR that removes all of them (but the intention is to get rid of them soon).

Reply:
@dndx Ok! Thanks for clarifying